app/Http/Requests/CategoryRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CategoryRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return false;
```

Comment:
```suggestion
        return true;
```
Nếu để authorize luôn trả về false thì làm gì có ai truy cập được nhỉ anh?

Reply:
u nhi quen mat